docs/asyncapi.yaml

```diff
@@ -62,85 +60,82 @@ components:
-        commsAddress:
-          $ref: '#/components/schemas/commsAddress'
+          $ref: "#/components/schemas/commsType"
+        commsAddresses:
+          $ref: "#/components/schemas/commsAddresses"
         personalisation:
-          $ref: '#/components/schemas/personalisation'
+          $ref: "#/components/schemas/personalisation"
         reference:
-          $ref: '#/components/schemas/reference'
+          $ref: "#/components/schemas/reference"
         oneClickUnsubscribeUrl:
-          $ref: '#/components/schemas/oneClickUnsubscribeUrl'
+          $ref: "#/components/schemas/oneClickUnsubscribeUrl"
         emailReplyToId:
-          $ref: '#/components/schemas/emailReplyToId'
+          $ref: "#/components/schemas/emailReplyToId"
 
     crn:
       type: number
-      description: Customer reference Number
+      description: Customer Reference Number (consists of 10 digits)
       minimum: 1050000000
       maximum: 9999999999
       example: 1234567890
 
     sbi:
       type: number
-      description: Single Business Identifier
+      description: Single Business Identifier (consists of 9 digits)
       minimum: 105000000
       maximum: 999999999
       example: 123456789
 
     sourceSystem:
       type: string
-      description: The name of the source system that originated the message
+      description: Name of the source system that originated the message
       example: "AHWP"
       minlength: 3
       maxlength: 100
       pattern: "^[a-z0-9-_]+$"
 
     notifyTemplateId:
       type: string
-      description: Notification template identifier
+      description: Notify template identifier
       example: "f33517ff-2a88-4f6e-b855-c550268ce08a"
       format: uuid
 
     commsType:
       type: string
-      description: Type of communication (e.g., email, SMS)
+      description: Type of communication (e.g. email, SMS)
       default: "email"
       example: "email"
       enum: ["email"]
 
-    commsAddress:
-      oneOf:
-        - type: string
-          description: The user's contact email address
-          example: "mrMegaFarmer@strawberryFortFarm.co.uk"
-          minLength: 1
-          maxLength: 320
-          pattern: "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}(\\.[a-zA-Z]{2,})?$"
-          title: accepts a single string
-        - type: array
-          title: accepts an array of strings
-          items:
-            type: string
-            description: A list of contact email addresses
-            example: "mrMegaFarmer@strawberryFortFarm.co.uk"
-            minLength: 1
-            maxLength: 320
-            pattern: "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}(\\.[a-zA-Z]{2,})?$"
+    commsAddresses:
+      type: array
+      title: Accepts an array of strings
+      description: List of contact email addresses (accepts a single email address)
+      example: ["mrMegaFarmer@strawberryFortFarm.co.uk", "msMegaFarmer@bluberryCottageFarm.com"]
+      minItems: 1
+      maxItems: 10
```

Comment:
For `maxItems` it is currently set to 10 but based on Abi's comment saying we don't need to cover this right now - should we leave `maxItems` as is or remove it completely and just have `minItems` set for the time being? Or based on your comment do we set it as 2 for the time being since you mentioned they typically send to 2 emails.